.gitignore

```diff
@@ -102,3 +102,6 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# samples
+samples/*
```

Comment:
there should be a newline at end of the file

Reply:
fixed